Makefile

```diff
@@ -67,28 +72,28 @@ dist: clean browserpass-linux64 browserpass-darwinx64 browserpass-openbsd64 brow
 
 .PHONY: install
 install:
-	install -Dm755 -t "$(DESTDIR)/usr/bin/" $(BIN)
-	install -Dm644 -t "$(DESTDIR)/usr/lib/browserpass/" Makefile
-	install -Dm644 -t "$(DESTDIR)/usr/share/licenses/browserpass/" LICENSE
-	install -Dm644 -t "$(DESTDIR)/usr/share/doc/browserpass/" README.md
+	install -Dm755 -t "$(BIN_DIR)/" $(BIN)
+	install -Dm644 -t "$(LIB_DIR)/browserpass/" Makefile
+	install -Dm644 -t "$(SHARE_DIR)/licenses/browserpass/" LICENSE
+	install -Dm644 -t "$(SHARE_DIR)/doc/browserpass/" README.md
 
-	install -Dm644 browser-files/chromium-host.json   "$(DESTDIR)/usr/lib/browserpass/hosts/chromium/$(APP_ID).json"
-	install -Dm644 browser-files/chromium-policy.json "$(DESTDIR)/usr/lib/browserpass/policies/chromium/$(APP_ID).json"
-	install -Dm644 browser-files/firefox-host.json    "$(DESTDIR)/usr/lib/browserpass/hosts/firefox/$(APP_ID).json"
+	install -Dm644 browser-files/chromium-host.json   "$(LIB_DIR)/browserpass/hosts/chromium/$(APP_ID).json"
+	install -Dm644 browser-files/chromium-policy.json "$(LIB_DIR)/browserpass/policies/chromium/$(APP_ID).json"
+	install -Dm644 browser-files/firefox-host.json    "$(LIB_DIR)/browserpass/hosts/firefox/$(APP_ID).json"
 
-	sed -i "s|%%replace%%|/usr/bin/$(BIN)|" "$(DESTDIR)/usr/lib/browserpass/hosts/chromium/$(APP_ID).json"
-	sed -i "s|%%replace%%|/usr/bin/$(BIN)|" "$(DESTDIR)/usr/lib/browserpass/hosts/firefox/$(APP_ID).json"
+	sed -i "s|%%replace%%|/usr/bin/$(BIN)|" "$(LIB_DIR)/browserpass/hosts/chromium/$(APP_ID).json"
+	sed -i "s|%%replace%%|/usr/bin/$(BIN)|" "$(LIB_DIR)/browserpass/hosts/firefox/$(APP_ID).json"
 
 # Browser-specific hosts targets
 
 .PHONY: hosts-chromium
 hosts-chromium:
 	@case $(OS) in \
 	Linux)      mkdir -p "/etc/chromium/native-messaging-hosts/"; \
-	            ln -sf "/usr/lib/browserpass/hosts/chromium/$(APP_ID).json" "/etc/chromium/native-messaging-hosts/" \
+	            ln -sf "$(LIB_DIR)/browserpass/hosts/chromium/$(APP_ID).json" "/etc/chromium/native-messaging-hosts/" \
```

Comment:
Do we also need to prefix the location of the link with some variables, i.e. "/etc/chromium/..." on this line?

Reply:
Nah I think this is fine, this is probably hardcoded in all browsers, and /etc is not a problem on Nix/NixOS.